alf/environments/carla_sensors.py

```diff
@@ -1307,8 +1308,10 @@ def is_running_red_light(self, actor):
             # red-light id encountered by the actor
             if np.any(same_lane):
                 encountered_red_light_id = self._traffic_light_actors[index].id
+                encountered_red_light_distance = dist[index]
 
-        return violated_red_light_id, encountered_red_light_id
+        return (violated_red_light_id, encountered_red_light_id,
```

Comment:
Also need to update `Returns` in docstring.

Reply:
Fixed